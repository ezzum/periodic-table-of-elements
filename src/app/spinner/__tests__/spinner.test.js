import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../spinner';

describe('Spinner tests', () => {
  const component = shallow(<Spinner></Spinner>);

  it('snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
