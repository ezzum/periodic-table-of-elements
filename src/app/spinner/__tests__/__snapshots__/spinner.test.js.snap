// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner tests snapshot 1`] = `ShallowWrapper {}`;
